simpledotorg/simple-server
# Change the order of treatment outcomes in cohort report

**Story card:** [sc-7527](https://app.shortcut.com/simpledotorg/story/7527/change-the-order-of-outcome-of-cohort-report)

## Because

Treatment outcomes in Cohort Report were not in the desired order
> Current Order - No BP Taken, BP not controlled, BP controlled & Missed visits
Desired Order - BP controlled, BP not controlled, Missed visit & No BP Taken.

## This addresses

Changes order of the treatment outcomes in the Cohort report. 
> The new order - BP controlled, BP not controlled, Missed visit & No BP Taken.

Before:
<img width="1132" alt="Before-Change" src="https://user-images.githubusercontent.com/32141642/158314572-1387cde7-e8c0-4329-9790-a4c231acd3f7.png">

After:
<img width="1149" alt="After-Change" src="https://user-images.githubusercontent.com/32141642/158314583-e2c7ffd8-e1b9-45db-a2fd-e2c22edf3b10.png">

## Test instructions

-  Check out to  branch `change-cohort-report-order`
-  Go to `Reports > Cohort Report` in Dashboard
-  Now you should be able to see the new order of outcomes in the cohort report
